region/crateria/west/Gauntlet Entrance.json

```diff
@@ -300,6 +319,24 @@
       ],
       "devNote": "TODO: There is a blue SpringBall strat to investigate."
     },
+    {
+      "link": [1, 2],
+      "name": "G-Mode Setup - Get hit by Waver, Using Power Bombs",
+      "notable": false,
+      "requires": [
+        "Morph",
+        {"ammo": { "type": "PowerBomb", "count": 3}}
+      ],
+      "exitCondition": {
+        "leaveWithGModeSetup": {}
+      },
+      "note": [
+        "This is harder when crossing the room and easier if the left door can be used to reset the room, but it will cost one extra Power Bomb.",
```

Comment:
```suggestion
        "This is harder when crossing the room and easier if the right door can be used to reset the room, but it will cost one extra Power Bomb.",
```